core/src/test/java/org/elasticsearch/monitor/os/OsProbeTests.java

```diff
@@ -32,16 +32,24 @@
 import static org.hamcrest.Matchers.lessThanOrEqualTo;
 
 public class OsProbeTests extends ESTestCase {
-    OsProbe probe = OsProbe.getInstance();
+    private final OsProbe probe = OsProbe.getInstance();
 
     public void testOsInfo() {
-        OsInfo info = probe.osInfo();
+        int allocatedProcessors = randomIntBetween(1, Runtime.getRuntime().availableProcessors());
+        long refreshInterval;
```

Comment:
I usually use a ternary for these sorts of random values. I don't usually like ternaries but for things like this I figure it is still pretty readable.